.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2.1
+
+commands:
+  nuodb_collector_command:
```

Comment:
What is the advantage of having a separate `command` vs having parametrized jobs?
For example, you can take a look at [go-test](https://github.com/nuodb/nuodb-helm-charts/blob/master/.circleci/config.yml) job.

Reply:
Added command just for reusability. The only difference between bare_metal and docker job is the environment variable. I set the variable in the job.